publiccode.yml

```diff
@@ -0,0 +1,108 @@
+# This repository adheres to the publiccode.yml standard by including this
+# metadata file that makes public software easily discoverable.
+# More info at https://github.com/italia/publiccode.yml
+
+publiccodeYmlVersion: '0.2'
+name: OpenProject
+applicationSuite: openDesk
+url: 'https://github.com/opf/openproject'
+roadmap: 'https://www.openproject.org/roadmap'
+releaseDate: '2025-01-21'
+softwareVersion: '15.1.1'
+developmentStatus: stable
+softwareType: standalone/web
+platforms:
+  - web
+usedBy:
+  - ZenDiS GmbH
+  - Stadt Köln
+  - Stadt Stuttgart
+  - Landtag Brandenburg
+  - Europäische Kommission
+  - Deutsche Bahn
+  - FITKO
+  - Stadt Chemnitz
+categories:
+  - agile-project-management
+  - budgeting
+  - enterprise-project-management
+  - help-desk
+  - knowledge-management
+  - project-collaboration
+  - project-management
+  - task-management
+  - time-management
+  - time-tracking
+  - workflow-management
+dependsOn:
+  open:
+    - name: PostgreSQL
+      versionMin: '13'
+maintenance:
+  type: contract
+  contacts:
+    - name: OpenProject GmbH
+      email: sales@openproject.com
+      phone: '+4928877707'
```

Comment:
According to the minimum version, this needs to list a contact like here: https://yml.publiccode.tools/example.html#minimum-version